To-DoList/app/src/main/java/com/ezequielc/to_dolist/DayAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.ezequielc.to_dolist;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+/**
+ * Created by student on 11/13/16.
+ */
+
+public class DayAdapter extends RecyclerView.Adapter<DayViewHolder> {
+    private ArrayList<Day> mDayArrayList;
+    private Context mContext;
+
+    public DayAdapter(Context context) {
+        mContext = context;
+        mDayArrayList = Singleton.getInstance().getDays();
+    }
+
+    @Override
+    public DayViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        return new DayViewHolder(LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.day_list, parent, false));
+    }
+
+    @Override
+    public void onBindViewHolder(final DayViewHolder holder, int position) {
+
+        final Day positionOfDay = mDayArrayList.get(position);
```

Comment:
A better name for this might be "currentDay" or something like that. Having position in the name implies that it is an index.